utils/tf.py

```diff
@@ -67,40 +72,52 @@ def __create_config(self, intra_threads: int, inter_threads=1):
         :return: TensorFlow config
         """
         config = tf.compat.v1.ConfigProto()
-        config.allow_soft_placement = True
```

Comment:
Breaks use of AML for GPU benchmarking

Reply:
its a remainder of my debugging, I restored it